test/functional/assets/pooltester.json

```diff
@@ -0,0 +1,216 @@
+{
+  "metadataVersion": "0.1.0",
+  "source": {
+    "hash": "0x2d1e1ed093dfb95c258342e80e7e1800c8c87cf4459837291a35245d23d15038",
+    "language": "ink! 3.0.0-rc4",
+    "compiler": "rustc 1.56.0-nightly"
+  },
+  "contract": {
+    "name": "pooltester",
+    "version": "0.1.0",
+    "authors": [
+      "RBB S.r.l"
+    ]
+  },
+  "spec": {
+    "constructors": [
+      {
+        "args": [
+          {
+            "name": "value",
+            "type": {
+              "displayName": [
+                "i64"
+              ],
+              "type": 1
+            }
+          }
+        ],
+        "docs": [],
+        "name": [
+          "new"
+        ],
+        "selector": "0x9bae9d5e"
+      },
+      {
+        "args": [],
+        "docs": [],
+        "name": [
+          "default"
+        ],
+        "selector": "0xed4b9d1b"
+      }
+    ],
+    "docs": [],
+    "events": [],
+    "messages": [
+      {
+        "args": [],
+        "docs": [],
+        "mutates": true,
+        "name": [
+          "get"
+        ],
+        "payable": false,
+        "returnType": {
+          "displayName": [
+            "i64"
+          ],
+          "type": 1
+        },
+        "selector": "0x2f865bd9"
+      },
+      {
+        "args": [],
+        "docs": [],
+        "mutates": true,
+        "name": [
+          "flip"
+        ],
+        "payable": false,
+        "returnType": null,
```

Comment:
return type is null?

Reply:
I think it means that the function doesn't return anything

Reply:
Alrighty. So more `void` than `NULL`. Doesn't python use `None` for that rather than `null`?

Reply:
Ignore me. I've just noticed this is json not python. I'd skipped to the next file without noticing due to GH's stupid design